packages/millicast-sdk/src/View.js

```diff
@@ -323,4 +373,104 @@ export default class View extends BaseWebRTC {
       })
     }
   }
-};
+
+  #getDRMConfiguration (mediaId) {
+    return this.#drmOptionsMap ? this.#drmOptionsMap.get(mediaId) : null
+  }
+
+  /**
+   * @typedef {Object} EncryptionParameters
+   * @property {String} keyId 16-byte KID, in lowercase hexadecimal without separators
+   * @property {String} iv 16-byte initialization vector, in lowercase hexadecimal without separators
+   * /
+
+  /**
+   * @typedef {Object} DRMOptions - the options for DRM playback
+   * @property {HTMLVideoElement} videoElement - the video HTML element
+   * @property {EncryptionParameters} videoEncParams - the video encryption parameters
+   * @property {String} videoMid - the video media ID of RTCRtpTransceiver
+   * @property {HTMLAudioElement} audioElement - the audio HTML audioElement
+   * @property {EncryptionParameters} [audioEncParams] - the audio encryption parameters
+   * @property {String} [audioMid] - the audio media ID of RTCRtpTransceiver
+   */
+
+  /**
+   * Configure DRM protected stream.
+   * When there are {@link EncryptionParameters} in the payload of 'active' broadcast event, this method should be called
+   * @param {DRMOptions} options - the options for DRM playback
+  */
+  configureDRM (options) {
```

Comment:
Question - should `options` be an object like this (or whatever variable names you like)- 

``` typescript
export interface DRMOptions {
 videoParams : VideoDRMOptions,
 audioParams : AudioDRMOptions
}

export interface VideoDRMOptions {
  videoElement : HTMLVideoElement,
  encryptionParams : EncryptionParameters,
  mid : String 
}

export interface AudioDRMOptions {
    audioElement : HTMLAudioElement,
    encryptionParams : EncryptionParameters,
    mid : String 
}
```

Reply:
I prefer flat structure which is easier to read and modify since this is not a complex one. Also in some cases, we only need 'audioElement' but no 'encryptionParams' and 'mid' for audio.

Reply:
Disagree but 👍